src/components/AssetSelection/AssetSelection.tsx

```diff
@@ -8,10 +8,8 @@ import { Text } from 'components/Text'
 import { selectAssets } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
-import { AssetChainDropdown } from './components/AssetChainDropdown/AssetChainDropdown'
-import { AssetMenuButton } from './components/AssetMenuButton'
-
-const disabledStyle = { cursor: 'not-allowed' }
+import { StyledAssetChainDropdown } from './components/AssetChainDropdown/AssetChainDropdown'
```

Comment:
q: what does `Styled` refer to across this PR?

Reply:
I'll follow up with you directly on this one as likely you've got some magic up your sleeve to make this tidy